src/App.tsx

```diff
@@ -1,223 +1,84 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import { getTodos } from './api/todos';
+
 import { AuthContext } from './components/Auth/AuthContext';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorNotification } from
+  './components/ErrorNotification/ErrorNotification';
+
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { filterTodosByCompleted } from './helpers/helpers';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [completedFilter, setCompletedFilter] = useState(FilterType.ALL);
+  const [errorMessage, setErrorMessage] = useState('');
 
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
-    if (newTodoField.current) {
-      newTodoField.current.focus();
+    if (user) {
+      getTodos(user.id)
+        .then(setTodos)
+        .catch(() => {
+          setErrorMessage('Can\'t load todos');
```

Comment:
Better to create ENUM for error texts
```suggestion
          setErrorMessage(TodoErrors.Load);
```